apps/docs-app/docs/experimental/sfc/index.md

```diff
@@ -0,0 +1,384 @@
+});
+```
+
+## IDE Support
+
+To support syntax highlighting and other IDE functionality with `.analog` files, you need to install an extension to support the format for:
+
+- [WebStorm 2024.1+ or IDEA Ultimate 2024.1+](https://github.com/analogjs/idea-plugin)
+
+> [Support for VSCode is coming! Please see this issue for more details](https://github.com/volarjs/angular-language-tools/).
+
+## Additional Configuration
+
+If you are using `.analog` files outside a project's root you will need to specify all paths of `.analog` files using globs, like so:
+
+```typescript
+export default defineConfig(({ mode }) => ({
+  // ...
+  plugins: [
+    analog({
+      experimental: {
+        supportAnalogFormat: {
+          include: ['/libs/shared/ui/**/*', '/libs/angularstart/ui/**/*'],
+        },
+      },
+    }),
+  ],
+}));
+```
+
+# Authoring an SFC
+
+Here's a demonstration of the Analog format building a simple counter:
+
+```html
+<script lang="ts">
+  // counter.analog
+  import { signal } from '@angular/core';
+
+  const count = signal(0);
+
+  function add() {
+    count.set(count() + 1);
+  }
+</script>
+
+<template>
+  <div class="container">
+    <button (click)="add()">{{count()}}</button>
+  </div>
+</template>
+
+<style>
+  .container {
+    display: flex;
+    justify-content: center;
+  }
+
+  button {
+    font-size: 2rem;
+    padding: 1rem 2rem;
+    border-radius: 0.5rem;
+    background-color: #f0f0f0;
+    border: 1px solid #ccc;
+  }
+</style>
+```
+
+See the [defineMetadata](#metadata) section for adding additional component metadata.
+
+# Metadata
+
+While class decorators are used to add metadata to a component or directive in the traditional Angular authoring methods, they're replaced in the Analog format with the `defineMetadata` global function:
+
+```typescript
+defineMetadata({
+  host: { class: 'block articles-toggle' },
+});
+```
+
+This supports all of the decorator properties of `@Component` or `@Directive` with a few exceptions.
+
+## Disallowed Metadata Properties
+
+The following properties are not allowed on the metadata fields:
+
+- `template`: Use the SFC `<template>` or `defineMetadata.templateUrl` instead
+- `standalone`: Always set to `true`
+- `changeDetection`: Always set to `OnPush`
+- `styles`: Use the SFC `<style>` tag
+- `outputs`: Implicitly added with `new EventEmitter()` usage
+- `inputs`: Use the `input` signal API instead
+
+# Using Components
+
+When using the Analog format, you do not need to explicitly export anything; the component is the default export of the `.analog` file:
+
+```typescript
+import { bootstrapApplication } from '@angular/platform-browser';
+import App from './app/app.analog';
+import { appConfig } from './app/app.config';
+
+bootstrapApplication(App, appConfig).catch((err) => console.error(err));
+```
+
+Further, when importing an `.analog` file into another `.analog` file, you do not need to add them to your `imports` like you do with non-analog components and directives.
```

Comment:
This needs to be changed/reworded as the functionality of automatically adding `.analog` to imports was removed in this PR: https://github.com/analogjs/analog/pull/905

The imports for LayoutFooter and LayoutHeader in the example below will also need to be updated - I'm not sure if it's a better idea to have them manually added to defineMetadata first, and then introduce import attributes later, or to have this example use import attributes right away. I lean toward manual import first, as it might better communicate what import attributes are actually doing.

Reply:
I'm not sure I understand 😓 Can you possibly make a commit to my branch/repo to fix this and I can review after?

Just so you can copy+paste quickly to do so:

```shell
gh pr checkout 987
```

Using [the GitHub CLI](https://cli.github.com/).

Reply:
Permission granted ;P :

https://github.com/crutchcorn/analog/invitations

(just sent you specifically an invite to my repo)

https://github.com/crutchcorn/analog/tree/beta-document-sfcs

Reply:
<img src="https://media2.giphy.com/media/dBGi39HzazuTV21S15/giphy.gif"/>

Reply:
@crutchcorn done 👍